README.md

```diff
@@ -128,51 +140,104 @@ print_endline "Wake Up";
   case were "distinct_id" is set after the mixpanel library has finish loaded \n
-  ```Shell
+  ```Ocaml
   Mixpanel.init ~token:"YOUR PROJECT TOKEN"
                 ~config:( Mixpanel.config
                           ~loaded: (function mixpanel ->
                                    distinct_id = mixpanel.get_distinct_id()))
                 ()
   ```
 
-  Source: [mixpanel API](https://developer.mixpanel.com/docs/javascript-full-api-reference)
+  Source: [Mixpanel API](https://developer.mixpanel.com/docs/javascript-full-api-reference)
 
-### Mixpanel.register
+### `Mixpanel.register`
 This function will add the properties given in argument to all futur
-"Mixpanel.tracks" raised in your application.
+`Mixpanel.tracks` raised in your application.
 For example:
-  ```Shell
+  ```Ocaml
   let plus = Mixpanel.Properties.create [("Regitered");(true)] in
   Mixpanel.register ~properties:plus ();
-  MMixpanel.track ~event:"Track" ~properties:(Mixpanel.Properties.create [("Gender", "Male");("Age", "21")]) ()
+  Mixpanel.track ~event:"Track" ~properties:(Mixpanel.Properties.create [("Gender", "Male");("Age", "21")]) ()
   ```
   That example will create a track called "Track" with the
   propertie "Registered:true" in addition to "Gender" and "Age"
 
-### Mixpanel.people_set
+### `Mixpanel.people_set`
 This funciton will execute the function "mixpanel.people.set" from the
 API. This function works a little bit like "track", in difference
 that the "properites" given in argument will insteade be added to the
 Mixpanel profil of the current user.
 
-### Mixpanel.identify
-[See mixpanel API](https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelidentify)
+### `Mixpanel.identify`
+> Identify a user with a unique ID to track activity across devices, tie
+a user to their events, and create a user profile. If you never call
+this method, unique visitors are tracked using a UUID generated the fist
+time they visit the site.
+
+[See Mixpanel API](https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelidentify)
 
-### Mixpanel.alias
+### `Mixpanel.alias`
 This function will add the "alias" string given in argument to the
 "alias name" list of the current user that can be seen on his profil on
 Mixpanel.
 
-### Mixpanel.reset
+### `Mixpanel.reset`
 
 > Clears super properties and generates a new random distinct_id for this instance. Useful for clearing data when a user logs out.
 
-Source : [mixpanel API](https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelreset)
+Source : [Mixpanel API](https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelreset)
+
+### `Mixpanel.set_group`
+> Register the current user into one/many groups.
+
+Source : [Mixapnel API](https://developer.mixpanel.com/docs/javascript-full-api-reference#mixpanelset_group)
+
+The second argument "group_ids" have to be a list (even if their is only
+1 element), in addition, even if the Mixpanel AST say that the list can
```

Comment:
```suggestion
one element), in addition, even if the Mixpanel AST says that the list can
```

Reply:
Mixpanel AST?

Reply:
I often got AST mixed up with API sorry, I will correct it.